File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbBrand;

import entry.PageResult;

public interface BrandService {
	
	public List<TbBrand> selectAll();
	
	public PageResult findPage(int pageNum,int pageSize);

	public void saveBrand(TbBrand brand);

	public void updateBrand(TbBrand brand);

	public TbBrand findById(Long id);

	public void deleteByIds(Long[] ids);

	public PageResult search(TbBrand brand, int page, int rows);

	public List<Map<String, String>> findBrandList();
}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
/**
 * 
 */
app.service('brandService',function($http){
		
		this.save = function(action,entry){
			return $http.post('../brand/' + action + '.do', entry);
		}
		
		this.findOne = function(id){
			return $http.get("../brand/findById.do?id=" + id);
		}
		
		this.delSelect = function(ids){
			return $http.get('../brand/delSelect.do?ids=' +ids);
		}
		
		this.search = function(page, rows,queryEntry){
			
			return $http.post('../brand/search.do?page=' + page + '&rows=' +rows, queryEntry);
		}
		
		this.findBrandList = function(){
			return $http.get('../brand/findBrandList.do');
		}
	});